roles/dbserver/tasks/postgresql.yml

```diff
@@ -16,6 +16,21 @@
 - include_role:
     name: geerlingguy.postgresql
     tasks_from: configure
+  vars:
+    postgresql_global_config_options:
```

Comment:
As discussed in Slack, we can adapt this PR and #403 to work with each other smoothly for both Zapier/Metabase type integrations as well as Datadog. :+1:

I'd prefer adding to the `postgres_global_config_options` variable by defining it in `all.yml`, as done in #403.